ironfish/src/network/peers/peer.ts

```diff
@@ -424,19 +397,6 @@ export class Peer {
 
     this._address = address
     this._port = port
-
-    if (address === null && port === null) {
```

Comment:
It looks like we've removed this logic but we aren't calling `PeerManager.getConnectionRetry.neverRetryConnecting` in place of removing this logic. Is that what we want to do? We call `setWebSocketAddress` in a few places where I don't think this logic has been replaced